### PR TITLE
Jramberger3/help request

### PR DESCRIPTION
**Pull request by JohnRamberger**
Most of the help request stuff

- Users can create help requests
- Request is then sent to firebase function that checks all available volunteers and picks the best
- Volunteer receives request and can accept/deny
- on accept, they are able to navigate to the request
- on deny, the user's request is blocked for the volunteer

### CODE REVIEW

**Review by medhanak — APPROVED** [review]

File: disaster_relief_aid_flutter/lib/component/HelpCallInProgressWrapper.dart [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  StreamSubscription<dynamic>? stream1;
+  StreamSubscription<dynamic>? stream2;
+
+  @override
+  void initState() {
+    // TODO: implement initState
+    super.initState();
+    stream1 =
+        VolunteeringSingleton().onHelpRequestAcceptedStream.listen((event) {
+      print("accepted help request");
```
Comment (kumeagidi) [inline disaster_relief_aid_flutter/lib/component/HelpCallInProgressWrapper.dart:30]: Remember to keep print statements out of finalized code.

**Review by kumeagidi — COMMENTED** [review]
Good work! Now a user can request help

**Review by acoustic-git-tar — APPROVED** [review]

File: disaster_relief_aid_flutter/lib/component/HelpCallInProgressWrapper.dart (hunk 1 above)
Comment (kozyjamal) [reply disaster_relief_aid_flutter/lib/component/HelpCallInProgressWrapper.dart:30]: i agree

Comment (JohnRamberger) [reply disaster_relief_aid_flutter/lib/component/HelpCallInProgressWrapper.dart]: I just removed this.

**Review by kozyjamal — APPROVED** [review]

**Review by Hchoudhry15 — APPROVED** [review]

**Review by kumeagidi — APPROVED** [review]
LGTM

**Pull request merged**